iothub/device/src/HsmAuthentication/HttpHsmSignatureProvider.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client.TransientFaultHandling;
+
+namespace Microsoft.Azure.Devices.Client.HsmAuthentication
+{
+    class HttpHsmSignatureProvider : ISignatureProvider
+    {
+        const string DefaultApiVersion = "2018-06-28";
```

Comment:
>DefaultApiVersion [](start = 21, length = 17)

Please take a look at the coding standards here:
[https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)

Specify visibility even if it is default.